x/auth/migrations/v6/migrate.go

```diff
@@ -0,0 +1,45 @@
+package v6
```

Comment:
This should be renamed to 5, same for the function. The v5 folder doc.go can be simply deleted (see my other comment)